massinstall/massinstall.go

```diff
@@ -154,6 +154,19 @@ func (mi *MassInstall) Run(md *model.SystemInstall, rootDir string, options args
 		}
 		if isAdvancedSelected {
 			log.Debug("Mass installer operating in Advanced Disk Partition Mode.")
+			var results []string
+			if md.IsDesktopInstall() {
```

Comment:
I am not sure why the check for IsDesktopInstall is required in massinstall.go? Am i missing something?

Reply:
IsDesktopInstall looks at the list of bundles to be installed. If there are desktop related bundles, we need much more disk space.
This is why there is a check for Desktop or Server, as that impacts the minimum required disk space needed for the installation regardless if we are installing from the CLI, TUI, or GUI.

Reply:
@mdhorn Thanks for explaining, maybe we need to change the name of the method itself
ContainsDesktopBundles() ?

Leave it upto you

Reply:
I'll rename it to IsTargetDesktopInstall.